<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>校验位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRateComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="dataBitsComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="parityComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Space</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="stopBitsComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="comRefreshButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comopen">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>状态：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_write">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_read">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>指令：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清零</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
